src/test/java/seedu/address/testutil/TypicalPersons.java

```diff
@@ -26,24 +26,14 @@
 public class TypicalPersons {
 
     public static final Person ALICE = new PersonBuilder().withName("Alice Pauline")
-            .withAddress("123, Jurong West Ave 6, #08-111")
-            .withEmail("alice@example.com")
-            .withPhone("94351253")
-            .withTags("car")
-            .withUpcoming("01-05-2024 1500").build();
+            .withAddress("123, Jurong West Ave 6, #08-111").withEmail("alice@example.com")
+            .withPhone("94351253").withTags("car").withUpcoming("01-05-2024 1500").build();
```

Comment:
I think better change the upcoming year to 2025. 1st May might cause issue if someone tested this on 1st may or after.

Reply:
Oh. This one is yc test, i tried changing the dates and it failed sth idk how to fix :(

Reply:
pass alr or still need me to fix @ziyang27

Reply:
still cant, paiseh 😭